scripts/subscribeEthereumDeposit.js

```diff
@@ -7,8 +7,9 @@ const mongoose = require('mongoose');
 const { BridgeClaim  } = require('../src/mongo/models');
 const ethers = require('ethers');
 const { curly } = require("node-libcurl");
-const { hexToU8a } = require("@cennznet/util");
+const { hexToU8a } = require("@polkadot/util");
 const pegAbi = require("../abi/ERC20Peg.json").abi;
+const PubSub = require('pubsub-js');
```

Comment:
how does pubsub js manage persistence? what happens to the queues if the service restarts?

Reply:
that's a good point.. we already are managing persistence via mongo db.. on restart, I will fill these queues with the data from db, (eth confirm/cennznet confirm) status in db, add to the queue